scripts/execute_ktlint_for_changed_modules.sh

```diff
@@ -0,0 +1,61 @@
+# This script first finds all the modules changed in a PR then executes ktlint on these modules.
+echo -----Fetching orign master
+git fetch origin master:refs/remotes/origin/master
+git branch -a
+echo -----Done fetching orign master
+
+echo -----Calculating modules to test
+# directory names that corresponds to a module that has unit tests - this list needs to be manually updated when a new module is added/deleted
+TESTABLE_MODULES="payments payments-core paymentsheet wechatpay link stripecardscan identity stripe-core payments-ui-core camera-core"
```

Comment:
what happens if we just run ktlint for all the modules?

Reply:
it's going to take longer time, please refer to the pr description for an example